opencti-platform/opencti-graphql/src/database/file-storage-helper.ts

```diff
@@ -133,6 +133,7 @@ export const deleteAllBucketContent = async (context: AuthContext, user: AuthUse
   }
 
   // Once all files are deleted, then bucket can be removed.
+  // -- commented because not working --
```

Comment:
do you have more details ? an issue link ?

Reply:
it's juste a precision regarding the commented line below (seen with @aHenryJard )